### PR TITLE
Check out what using rubocop-rspec does

### PR DESCRIPTION
**Pull request by jyaworski**
@bastelfreak @rnelson0

### CODE REVIEW

**Comment by jyaworski** [comment]
OK. @bastelfreak @rnelson0 @nibalizer

This is what autocorrect gave me. Do we want to disable the name cop?

**Comment by rnelson0** [comment]
It looks okay to me, but I definitely have opinions on word tenses in commit messages, for instance.

**Comment by jyaworski** [comment]
@rnelson0 it specifically says something like 'do not use should to describe'....

Any opinions on disabling cops?

File: spec/classes/collectd_init_spec.rb [hunk 1]
```diff
@@ -32,7 +32,7 @@
         { package_install_options: 'not_an_array' }
       end
 
-      it 'should fail' do
+      it 'fails' do
```
Comment (igalic) [inline spec/classes/collectd_init_spec.rb]: i like the optimism in these changes ;)

Comment (jyaworski) [inline spec/classes/collectd_init_spec.rb]: I do too. I think we should enable this.

**Pull request closed without merging**